src/pages/index.js

```diff
@@ -4,15 +4,17 @@ import Card from '../components/Card';
 import Layout from '../components/Layout';
 import {db} from '../db';
 
+//import Effects from './effects';
+
 export default function HomePage() {
 	return (
 		<Layout>
 			<Head>
-				<title key="title">My Project</title>
+				<title key="title"></title>
```

Comment:
Füge hier bitten den Titel hinzu z.B. Asana App 😁 
```suggestion
				<title key="title">Asana App</title>
```

Reply:
Ok, erledigt . "Yoga App"

Reply:
Ok, erledigt.  "Yoga App"